### PR TITLE
Use expanded links instead of expanding the links in the content_store

### PR DESCRIPTION
**Pull request by dougdroper**
Travel advice document_types still need to go through the old flow

### CODE REVIEW

File: app/presenters/content_item_presenter.rb [hunk 1]
```diff
@@ -42,6 +42,7 @@ def as_json(options = nil)
 private
 
   def links
+    return @item.expanded_links unless document_type_expanded?
```
Comment (danielroseman) [inline app/presenters/content_item_presenter.rb]: Should we remove `expanded_links` from the PUBLIC_ATTRIBUTES above?

Comment (dougdroper) [inline app/presenters/content_item_presenter.rb]: Yes definitely, updated.

File: app/presenters/content_item_presenter.rb [hunk 2]
```diff
@@ -52,4 +52,8 @@ def links
   def present_linked_item(linked_item)
     LinkedItemPresenter.new(linked_item, @api_url_method).present
   end
+
+  def document_type_expanded?
+    @item.document_type =~ /travel_advice/
```
Comment (gpeng) [inline app/presenters/content_item_presenter.rb]: `attr_reader :item, :api_url_method` would avoid accessing the instance variables directly

Comment (dougdroper) [inline app/presenters/content_item_presenter.rb]: thanks for the handy tip 👍  updated

**Pull request merged**